pkg/authorization/authorizer/scope/converter.go

```diff
@@ -135,6 +135,28 @@ const (
 	UserFull = UserIndicator + "full"
 )
 
+var defaultSupportedScopesMap = map[string]string{
+	UserInfo:               "Read-only access to your user information (including username, identities, and group membership)",
+	UserAccessCheck:        `Read-only access to view your privileges (for example, "can I create builds?")`,
+	UserListScopedProjects: `Read-only access to list your projects viewable with this token and view their metadata (display name, description, etc.)`,
+	UserListAllProjects:    `Read-only access to list your projects and view their metadata (display name, description, etc.)`,
+	UserFull:               `Full read/write access with all of your permissions`,
+}
+
+func DefaultSupportedScopes() []string {
```

Comment:
We need this to have a consistent order so use `return sets.StringKeySet(defaultSupportedScopesMap).List()` (you can store this as a variable if you want to).

`sets` = `"k8s.io/apimachinery/pkg/util/sets"`